unit_test/option_prepare_test.py

```diff
@@ -21,6 +21,7 @@
     "cp310",
     "cp311",
     "cp312",
+    "cp313",
```

Comment:
Does both `"cp313"` and `"cp313t"` need to be added here and below?

Reply:
if we want to add specific tests for free threaded python, let's do that in a following PR.